test/play_store_test.dart

```diff
@@ -5,38 +5,35 @@
 import 'package:flutter_test/flutter_test.dart';
 import 'package:upgrader/upgrader.dart';
 
-import 'mock_play_store_client.dart';
-
 void main() {
-  var androidId = 'com.kotoko.express';
+  var applicationId = 'com.kotoko.express';
   test('testing PlayStoreSearchAPI properties', () async {
-    final playStore = PlayStroeSearchApi();
+    final playStore = PlayStroeSearchAPI();
     expect(playStore.debugEnabled, equals(false));
     playStore.debugEnabled = true;
     expect(playStore.debugEnabled, equals(true));
     expect(playStore.playStorePrefixURL.length, greaterThan(0));
 
     expect(
-        playStore.lookupURLById(androidId),
+        playStore.lookupURLById(applicationId),
         equals(Uri.https(
-            'play.google.com', '/store/apps/details', {'id': androidId})));
+            'play.google.com', '/store/apps/details', {'id': applicationId})));
   });
 
   test('testing lookupById', () async {
-    final client = MockPlayStoreSearchClient.setupMockClient();
```

Comment:
Why did you remove the use of `MockPlayStoreSearchClient ` in this test?

Reply:
Ok